### PR TITLE
021 kb 32

### PR DESCRIPTION
**Pull request by mashakeita**
Ticket #021KB-32
-Add a footer 
-Added styling for header

### CODE REVIEW

File: client/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import '../Footer/Footer.scss'
+import React from 'react';
+
+const Footer = () => {
+    return (
+        <footer className='footer'>
+            © InStock Inc. All Rights Reserved.
```
Comment (nilouhou) [inline client/src/components/Footer/Footer.jsx:7]: @mashakeita what do you think to have <p className ="body-small">...</p>
then you don't need to import typography, and @include body-sm-mobile; then in tablet view, just overwrite it with   @include tablet {
    @include body-sm-tablet;
  }

Comment (nilouhou) [reply client/src/components/Footer/Footer.jsx:7]: `<p className="body-small>...</p>`

File: client/src/components/Footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  font-family: "TitilliumWeb";
+  @include mobile {
+    padding: 2.5rem 0rem;
+  }
+  @include tablet {
+    @include body-sm-tablet;
+  }
+  @include desktop {
+    padding: 3rem 0rem;
+    @include body-sm-tablet;
```
Comment (manecipoglu) [inline client/src/components/Footer/Footer.scss:17]: You already have this styling for tablet. You do not need to write it again for the desktop view. Everything you put inside the desktop view is added what is already on the tablet view.

File: client/src/components/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@use "../../styles/partials/variables" as *;
+@use "../../styles/partials/mixins" as *;
+@use "../../styles/partials/typography" as *;
+
+.footer {
+  color: $secondary-dark;
+  background-color: $background-light;
+  font-family: "TitilliumWeb";
```
Comment (manecipoglu) [inline client/src/components/Footer/Footer.scss:8]: I believe we have the font-family defined on the body element styles. Can you confirm that the fonts do not change if you erase the font-family line here?

Comment (shaynelar) [reply client/src/components/Footer/Footer.scss:8]: Was also going to point this out

File: client/src/App.js [hunk 4]
```diff
@@ -40,7 +42,8 @@ function App() {
 						render={(props) => <div>Inventory Item Add</div>}
 					/>
 				</Switch>
+			<Footer/>
 			</BrowserRouter>
-	);
```
Comment (shaynelar) [inline client/src/App.js:44]: App.js is not really a page here so it doesn't need a footer. ITs just being used as source of truth for our routing. Footer will go on the pages that live at the routes

**Pull request merged**